src/helpers/fujiAPI.ts

```diff
@@ -15,22 +15,35 @@ export async function getFUJIResults(studyInfo: StudyInfo): Promise<JSON | strin
   let fujiRes: AxiosResponse<any, any>;
   let fujiResults: any | string;
   let retries: number = 0;
+  //
+  let data = JSON.stringify({
+  "metadata_service_endpoint": "",
+  "metadata_service_type": "oai_pmh",
+  "object_identifier": studyInfo.url,
+  "metric_version": "metrics_v0.8",
+  "test_debug": true,
+  "use_datacite": true,
+  "auth_token": base64UsernamePassword,
+  "auth_token_type": "Basic"
+});
+
+let config = {
+  method: 'post',
+  maxBodyLength: Infinity,
+  url: fujiEndpoint,
+  headers: { 
+    'Content-Type': 'application/json',
+    'Accept': '*/*',
+  },
+  auth: {
+    username: fujiUsername,
+    password: fujiPassword
+  },
+  data : data
+};
```

Comment:
```suggestion
  //
  const data = JSON.stringify({
    "metadata_service_endpoint": "",
    "metadata_service_type": "oai_pmh",
    "object_identifier": studyInfo.url,
    "metric_version": "metrics_v0.8",
    "test_debug": true,
    "use_datacite": true,
    "auth_token": base64UsernamePassword,
    "auth_token_type": "Basic"
  });

  const config: AxiosRequestConfig<string> = {
    method: 'post',
    maxBodyLength: Infinity,
    url: fujiEndpoint,
    headers: {
      'Content-Type': 'application/json',
      'Accept': '*/*',
    },
    auth: {
      username: fujiUsername,
      password: fujiPassword
    },
    data: data
  };
```

Use `const`, formatting consistency